client/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "monaco-languageclient",
-    "version": "0.10.2",
+    "version": "0.10.3",
```

Comment:
please don't do it manually, version is updated automatically during the release

Reply:
Does this also count for the version number used by the examples?

Reply:
I'll just put all the versions for the languageclient back to 0.10.2 and leave it up to the merger / the release script to bump them.